### PR TITLE
Adds attack logs for golems being summoned.

### PR DESCRIPTION
**Pull request by FlattestGuitar**
:cl: FlattestGuitar
tweak: golems now have their master saved in attack logs
/:cl:

Easier to check attack logs than ask around for a golem's master in case they run around the brig chainsawing sec in half for no apparent reason.

Remember, kids, traitor responsibly.

### CODE REVIEW

File: code/modules/research/xenobiology/xenobiology.dm [hunk 1]
```diff
@@ -215,6 +215,7 @@
 		SM.faction = user.faction
 		SM.master_commander = user
```
Comment (FalseIncarnate) [inline code/modules/research/xenobiology/xenobiology.dm]: This already logs the master of the sentient animal in a var which any admin can easily check. Adding the same thing to the attack_log seems somewhat redundant.

Also, if a sentient animal manages to reproduce, any offspring inherit the master_commander value, but wouldn't have the new attack log added below.

Comment (Fox-McCloud) [inline code/modules/research/xenobiology/xenobiology.dm]: Yup, what False said; this really only needs to be done for golems.

Comment (FlattestGuitar) [inline code/modules/research/xenobiology/xenobiology.dm]: Ah, I didn't consider inheritance a thing. It's only on golems now.

**Comment by Fox-McCloud** [comment]
looking into this further, this would be better to use `add_logs` to be honest.

**Comment by FlattestGuitar** [comment]
Nice one, me.

**Pull request merged**